tests/integration/v4_slowquery_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
```

Comment:
It would be great if your test works for all 4.0+ versions. Otherwise it is too horrible to write one file for one db version.